packages/vue/src/cypress/FTableColumn.pageobject.ts

```diff
@@ -14,14 +15,23 @@ export class FTableColumnPageObject implements BasePageObject {
         this.el = () => cy.get(this.selector);
     }
 
+    /**
+     * @deprecated Use ´FInteractiveTablePageObject.cell()´ instead. Deprecated since 6.11.0.
+     */
     public tableRowBodyContent(position: number): DefaultCypressChainable {
         return cy.get(`${this.selector} td:nth(${position})`);
     }
 
+    /**
+     * @deprecated Use ´FInteractiveTablePageObject.header()´ instead. Deprecated since 6.11.0.
+     */
     public tableRowHeaderContent(): DefaultCypressChainable {
         return cy.get(`${this.selector} th`);
     }
 
+    /**
+     * @deprecated Use ´FInteractiveTablePageObject.checkbox()´ instead. Deprecated since 6.11.0.
```

Comment:
Mergade #532, du kanske kan vara försökskanin?

```suggestion
     * @deprecated Use ´FInteractiveTablePageObject.checkbox()´ instead. Deprecated since %version%.
```

Så slipper du tänka på version mer.

Reply:
Ok, nice. Jag provar.